L/libjulia/common.jl

```diff
@@ -139,6 +140,11 @@ function configure(version)
     EOM
     fi
 
+    if [[ "${target}" == arm-* ]]; then
+        echo "MARCH=armv7-a" >>Make.user
+    fi
```

Comment:
Better to check `bb_full_target`; we want to disambiguate between `armv7l` and `armv6l`.

Reply:
I wasn't even aware that there are also armv6l builds possible, all I've been seeing was/is armv7l (but I just now saw that GMP_jll indeed has both). I have no idea what happens if one tries to build libjulia_jll for armv6l. So I will tighten this to something like `${bb_full_target} == armv7l-*`

Reply:
Yeah, I've been trying to sneakily improve armv6l in the background for a while, and we're pretty darn close now.  It's possible we might actually have an official armv6l build of 1.6, but no rush to include it here yet.  (It's not given back from `supported_platforms()` unless you pass in the `experimental=true` kwarg)

Reply:
Done. Also added a comment before the `if` explaining what this is good for, so that future generations don't have to suffer quite as much as their predecessors ;-)